### PR TITLE
Completed es6-destructuring exercise.

### PR DESCRIPTION
**Pull request by saraysc**
image

### CODE REVIEW

File: es6-destructuring/main.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    libraryID: 3245
+  },
+  {
+    title: "Dead Dead Demon's De De De De Destruction",
+    author: 'Inio Asano',
+    libraryID: 1233
+  }
+];
+
+const [book3, book4, book5] = [library[0], library[1], library[2]];
```
Comment (Zhiwutian) [inline es6-destructuring/main.js:43]: on the right side of this `=` you should just have `library`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one update needed

**Comment by saraysc** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**